web/src/i18n/index.ts

```diff
@@ -4,7 +4,7 @@ import en from "./translations/en.json"
 
 import type { I18n, I18nOptions, Locale, Composer } from "vue-i18n"
 
-export const SUPPORT_LOCALES = ["en", "no"]
+export const SUPPORT_LOCALES = ["en", "no", "fr", "zh"]
```

Comment:
Remove unsupported locales from this array, so you don't need to catch it later.